src/test/java/one/util/streamex/api/EntryStreamTest.java

```diff
@@ -86,10 +87,11 @@ private static void checkAsString(String expected, EntryStream<?, ?> stream) {
     @Test
     public void testCreate() {
         assertEquals(0, EntryStream.empty().count());
-        assertEquals(0, EntryStream.empty().count());
```

Comment:
The duplication here was intentional and it actually caught a bug in the early version of StreamEx when `EntryStream.emtpy()` returned a shared object. Please restore it.